speech/unit_tests/test_client.py

```diff
@@ -167,11 +189,30 @@ def test_sync_recognize_with_empty_results(self):
         client = self._makeOne(credentials=credentials)
         client.connection = _Connection(SYNC_RECOGNIZE_EMPTY_RESPONSE)
 
-        with self.assertRaises(ValueError):
-            sample = Sample(source_uri=self.AUDIO_SOURCE_URI,
-                            encoding=speech.Encoding.FLAC,
-                            sample_rate=self.SAMPLE_RATE)
-            client.sync_recognize(sample)
+        with self.assertRaises(IndexError):
```

Comment:
I'm not sure which is correct I guess. I was thinking about it and it seemed like IndexError would be the right one but I guess an argument for ValueError could be made too?